dev-games/aseprite/files/aseprite-1.3.5_laf-strict-aliasing.patch

```diff
@@ -0,0 +1,149 @@
+--- a/laf/base/cfile.cpp
```

Comment:
Is this upstream somewhre? If so, include a reference at the top? If not, why not?

Reply:
ping

Reply:
Proposed upstream fix https://github.com/aseprite/laf/pull/84

Reply:
please link to it above the patch, and git format-patch the patch properly (if git).  see, for instance, https://gitweb.gentoo.org/repo/gentoo.git/tree/app-emulation/qemu/files/qemu-7.2.0-linux-headers-6.2-glibc-2.36.patch

Reply:
Done.